This patch is unnecessary for raptor-2.0.5 and above, only to be used with 2.0.4

http://bugs.gentoo.org/show_bug.cgi?id=377253#c12

--- a/configure.ac	2011-09-10 22:31:37.791042002 -0700
+++ b/configure.ac	2011-09-10 22:51:12.789352501 -0700
@@ -407,8 +407,12 @@
 #ifdef HAVE_MATH_H
 #include <math.h>
 #endif
+#ifdef HAVE_STDLIB_H
+#include <stdlib.h>
+#endif
 ]], [[
-double d = trunc(1.0F) + round(1.0F);
+double t = atof("0.000");
+return (int)(trunc(t) * round(t));
 ]])],
    AC_MSG_RESULT(yes)
    trunc_round_lib=libc,
@@ -421,8 +425,12 @@
 #ifdef HAVE_MATH_H
 #include <math.h>
 #endif
+#ifdef HAVE_STDLIB_H
+#include <stdlib.h>
+#endif
 ]], [[
-double d = trunc(1.0F) + round(1.0F);
+double t = atof("0.000");
+return (int)(trunc(t) * round(t));
 ]])],
      AC_MSG_RESULT(yes)
      need_libm=1
